rpcs3/rpcs3qt/pad_settings_dialog.ui

```diff
@@ -997,7 +997,7 @@
        </layout>
       </item>
       <item>
-       <layout class="QVBoxLayout" name="verticalLayout_middle">
+       <layout class="QVBoxLayout" name="verticalLayout_middle" stretch="0,0,0,0,0,1">
```

Comment:
Why the stretch?

Reply:
Without it the description box would always shrink to size of contents.

Reply:
That's why we put extra newlines into the tooltips.

Reply:
I put some (well, br tags but they're the same thing) into the default text but the text box still shrunk down without the stretch rule.